orderbook/openapi.yml

```diff
@@ -296,6 +296,84 @@ paths:
           description: Token non-existent or no valid price found
         500:
           description: Unexpected internal error while processing the request
+  /api/v1/fee_and_quote/sell:
```

Comment:
I'm still not excited about the name, although I don't consider super important, and since there's many approvals already I assume people is happy with it, so I won't oppose

I'm approving, although leaving my thinking so it can get lost in this PR comments :) 
- why snake case? I thought we were using kebab 🍗
- I liked more how it was before, base-quote (in terms of a market), although i see for example paraswap don't use the market notation. The reason i like market notation is because for me is closer to what a user does: you are in WETH-DAI market, and you want to buy or sell WETH.
- Even if we don't like the market notation, and we want to use sell/buy token notation because it's closer to the contracts, I would prefer a simpler name `prices/sell`. For me, the fact that we return also the fee shouldn't have so much relevance to change the endpoint name. The name doesn't need to be an exaustive description of all the data. The main thing we are querying is the price.

Reply:
1. We are using kebab case for the types but hasn't come up yet for routes because the rest are one word. Actually, there is solvable_orders which is snake case but I don't think we consciously thought about that. I will change this PR to use kebab case for the route too.
2. Personally I find calling things sell_amount and buy_amount more obvious than quote and base. But not sure how normal this is. I'm also not sure the market abstraction is that useful because the price and fee in one direction can be (slightly) different than in the other direction.
3. Right now we have a fee route and a markets route so I wanted this name to show this is route is different. Maybe once it is clear which routes will stay we can rename this one and think about what the best name would be. Like you said I also don't find it super important and don't want to spend too long discussing here.

Reply:
> Personally I find calling things sell_amount and buy_amount more obvious than quote and base

I think this is fine. We used "markets" in GPv1 for price estimation and it was a pretty large source of confusion. I think making it "sell" and "buy" might be weird to the financially inclined, but at least to me it feels less ambiguous and you can immediately populate request data without having to think too much.

> I will change this PR to use kebab case for the route too.

Personally, I prefer kebab case for URLs. I don't know why, nor do I have a good reason for it.

Reply:
All good 👌
Thxxx